flink-streaming-scala/src/test/scala/org/apache/flink/streaming/api/scala/AsyncDataStreamITCase.scala

```diff
@@ -88,31 +89,16 @@ class AsyncDataStreamITCase extends AbstractTestBase {
   }
 
   @Test
-  def testOrderedWaitUsingAnonymousFunction(): Unit = {
-    testAsyncWaitUsingAnonymousFunction(true)
-  }
-
-  @Test
-  def testUnorderedWaitUsingAnonymousFunction(): Unit = {
-    testAsyncWaitUsingAnonymousFunction(false)
+  def testOrderedWait(): Unit = {
```

Comment:
Please revert all the renamings to their previous names (`testOrderedWaitUsingAnonymousFunction`, `testUnorderedWaitUsingAnonymousFunction`, and `testAsyncWaitUsingAnonymousFunction`), as refactorings should not be part of this PR (they are not relevant to the attached JIRA issue) and also I think that, although not perfect, the previous names were giving more information about what the tests are trying to do.